gflownet/utils/batch.py

```diff
@@ -1062,6 +1066,48 @@ def is_valid(self) -> bool:
             return False
         return True
 
+    def traj_indices_are_consecutive(self) -> bool:
```

Comment:
This function does not check whether indices are going in the sorted order, only that the consecutive indices are present in the `self.trajectories `dict because `set` object does not care about the order (e.g. `set([1,2,3]) == set([3,1,2])` is `True`). Is that what you expect here or the order should matter (from the docstring it seems to me like you want to check the order too)

Reply:
Oops you're right! Fixed [here](https://github.com/alexhernandezgarcia/gflownet/pull/167/commits/01a605e693e863b7c636040710f0d020debd4633).